stylesheets/_get-started.scss

```diff
@@ -272,3 +272,351 @@ ul.toc {
 
 #new-rating, #your-rating, #avg-rating {display:none;}
 #rating-section {padding-top:1em;}
+
+// NEW GET STARTED - BLUE BOXES STYLES
+.multipath-tabs-sm {
+  margin-bottom: 15px;
+  padding-left: 0;
+  padding-bottom: 12px;
+  border-bottom: 1px solid #BDBDBD;
+  ul {
+    li:before {
+      display: none;
+    }
+    li {
+      max-width: 100px;
+      display: inline;
+      margin-right: 10px;
+      padding: 15px 20px 15px 10px;
+      &.active {
+        border: 1px solid #BDBDBD;
+        border-bottom-color: $white;
+        a {
+          padding-bottom: 4px;
+          border-bottom: 3px solid $red;
+        }
+      }
+      a {
+        font-size: 1em;
+      }
+    }
+  }
+}
+
+#get-started h2{
```

Comment:
Using IDs, and to a lesser extend tag names, is often considered bad practice in css when you can get by with a class. I'v also found it difficult on the drupal side when the css heavily depends on the html structure to recreate things correctly.